README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# HW 01
+
+### Introduction
+
+![-](https://user-images.githubusercontent.com/67297411/85501690-7d090680-b5ab-11ea-9c45-1364676d9eda.png)
```

Comment:
This didn't quite get inserted correctly. The way you normally want to do it is by uploading the image to RStudio, then commit that change to GitHub. Here, we don't have the file locally in the repo which means this is less reproducible than it could be otherwise.